blockbase/theme.json

```diff
@@ -31,29 +31,40 @@
 			"gradients": [],
 			"palette": [
 				{
-					"slug": "black",
+					"slug": "primary",
 					"color": "#000000",
-					"name": "Black"
+					"color-value-i-would-like-to-use": "var(--wp--custom--color--black)",
```

Comment:
Because the color picker assumes a color value rather than a css variable this doesn't work as desired.  I THINK I saw some shorthand in other places to use values like these in a way that might make this work. 

Regardless the "custom" colors are where the "descriptive" colors live, though they aren't very useful right now.